README.md

```diff
@@ -82,7 +82,7 @@ See [Workflows](./docs/workflows.md) for more information.
 
 ## License
 
-The source code in this repository is licensed under any of the following at your option:
```

Comment:
Not needed

Reply:
I added it be explicit, as that was a worry in https://github.com/bevyengine/bevy_github_ci_template/issues/45

Reply:
Okay, my concern is that the wording is awkward. Would moving the `Credits` section to before `License` work?

Reply:
That's a good idea.

Reply:
[`7cca212` (#173)](https://github.com/TheBevyFlock/bevy_quickstart/pull/173/commits/7cca212935674a4bc7917759bba970a997a0ca7a)

Reply:
I still believe "The source code—but not the assets—in this repository is licensed ..." is redundant and makes the sentence read weird: "source code is" but "assets are", so they don't align grammatically.

The sentence already intentionally says "The source code in this repository is licensed" instead of just "This repository is licensed" to make it explicit.

Reply:
If you strongly disagree, I can approve anyways. Rest of the PR is fine.

Reply:
I have no strong feelings at all, I just wanted to address the worries of those who have thought about this issue before. I'll revert it and send licensing people your way :P

Reply:
[`8f2af1b` (#173)](https://github.com/TheBevyFlock/bevy_quickstart/pull/173/commits/8f2af1bff2965be239c6253c7c3350c13d64f807)